### ISSUE TITLE
[Designer] Don't use placeholder for generated Inputs

### ISSUE THREAD

**Issue by golddove** [microsoft/AdaptiveCards#4768]
# Version of SDK

latest main

# Details

The designer, when you drag in most inputs (e.g. `Input.ChoiceSet`, `Input.Text`) generates elements with the `placeholder` property set. As we no longer want to encourage placeholders, we shouldn't generate them by default.

**Comment by shalinijoshi19** [comment]
@matthidinger / @dclaux / @RebeccaAnne what was our final word on placeholder support - I believe we were going to deprecate them eventually? @almedina-ms this was one of the discussions you'd been driving -- want to pick this up to conclusion?

**Comment by dclaux** [comment]
@golddove I don't think we decided to deprecate placeholders. We have agreed that they are not a solution as far as accessibility is concerned, but they are useful nevertheless.

**Comment by golddove** [comment]
@dclaux to clarify, I wasn't suggesting to remove placeholder support. This is to modify the default element generated by the designer when creating a new input. By default, we should use the recommended property.

If there is concern about hosts not yet supporting 1.3, then this change can wait along with #4612.

**Comment by dclaux** [comment]
@golddove got it, it makes sense. When we do this we should also have the designer set the `label` property to a default value.
